eng/common/scripts/Helpers/Metadata-Helpers.ps1

```diff
@@ -129,18 +129,8 @@ function GenerateDocsMsMetadata($originalMetadata, $language, $languageDisplayNa
     $msauthor = $author
   }
   $date = Get-Date -Format "MM/dd/yyyy"
-
-  $metadataTable = @{
```

Comment:
Have you considered [Ordered] here? https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_hash_tables?view=powershell-7.2#creating-ordered-dictionaries

Reply:
I don't want to order by alphabet. I want to have custom defined order.

Reply:
That is what the [ordered] tag does.